apps/web/next.config.mjs

```diff
@@ -12,7 +12,6 @@ const INTERNAL_PACKAGES = [
 const nextConfig = {
   output: 'standalone',
   transpilePackages: INTERNAL_PACKAGES,
-  serverExternalPackages: ['@sentry/nextjs', 'bullmq', 'jose'],
```

Comment:
removed these as I'm not sure they are needed

Reply:
Why not. Otherwise, they're packaged with next

Reply:
i think that's fine? it's backend code so no penalty for end user and i think the nextjs compiler treeshakes what is not needed from those dependencies

Reply:
i'm gonna keep it as it was just to not touch too many knobs at once